include/TestEnumerations.h

```diff
@@ -79,12 +79,12 @@ namespace Mezzanine
         SUPPRESS_CLANG_WARNING("-Wglobal-constructors")
             /// @brief Corresponds to TestResult::Success
             const Mezzanine::String SuccessString("Success");
-            /// @brief Corresponds to TestResult::Warning
-            const Mezzanine::String WarningString("Warning");
             /// @brief Corresponds to TestResult::Skipped
             const Mezzanine::String SkippedString("Skipped");
             /// @brief Corresponds to TestResult::Cancelled
             const Mezzanine::String CancelledString("Cancelled");
+            /// @brief Corresponds to TestResult::Warning
+            const Mezzanine::String WarningString("Warning");
             /// @brief Corresponds to TestResult::Inconclusive
             const Mezzanine::String InconclusiveString("Inconclusive");
```

Comment:
Should these lines have StringView instead of String as types?

Reply:
Fixed and I adjusted the box versions to be like this as well. No need to do a bunch of allocations for simple test results.